File: tuts/005-cloudfront-gettingstarted/php/CloudFrontWrapper.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

namespace Example\CloudFront;

use Aws\CloudFront\CloudFrontClient;
use Aws\Exception\AwsException;

class CloudFrontWrapper
{
    private CloudFrontClient $client;

    public function __construct(CloudFrontClient $client)
    {
        $this->client = $client;
    }

    // TODO: Add wrapper methods matching CLI tutorial actions
}

File: tuts/005-cloudfront-gettingstarted/ruby/cloudfront_wrapper.rb
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

require 'aws-sdk-cloudfront'
require 'logger'

class CloudFrontWrapper
  def initialize(client, logger: Logger.new($stdout))
    @client = client
    @logger = logger
  end

  # TODO: Add wrapper methods matching CLI tutorial actions
end
